Formula/threesdk.rb

```diff
@@ -0,0 +1,140 @@
+class Threesdk < Formula
+    desc "Software development kit for JumpscaleX"
+    homepage "https://sdk.threefold.io/"
+    url "https://github.com/threefoldtech/jumpscaleX_core/archive/v10.5.2.tar.gz"
+    sha256 "d2ce77a61a198b2cd53f4acce5fd7e5c36c369fc83a05b9f74314530b4f4a175"
+
+    depends_on "python@3.8" => :build
+    depends_on "gnu-sed" => :build
+    depends_on "upx" => :build
+
+    resource "docopt" do
```

Comment:
It doesn't install any of these

Reply:
sorry didn't get that, we need upx, but gnu-sed can be removed, That what you mean ?

Reply:
None of the resources are installed

Reply:
installed now using virual env